iDiff/differs/pipDiff.go

```diff
@@ -86,11 +81,30 @@ func (d PipDiffer) getPackages(path string) (map[string]utils.PackageInfo, error
 					glog.Errorf("Could not find Python package %s for corresponding metadata info", packageName)
 					continue
 				}
+				currPackage := utils.PackageInfo{Version: version, Size: size}
+				addToMap(packages, packageName, pyVersion, currPackage)
+			}
 
-				packages[packageName] = utils.PackageInfo{Version: version, Size: size}
+			// if not package, check if Python file
```

Comment:
I don't think you need this anymore, now that I changed how we were matching packages' dist-infos to the actual packages/scripts above with indexing